ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.txt

```diff
@@ -130,14 +130,14 @@ hideall "--Reset--"
 
 
 ### Phase 4: Snacks ###
-800 "Freefire" sync /:Titan:2CF5:/ window 800,0
+800 "Freefire" sync /:The Ultima Weapon:2CF5:/ window 800,0
 815 "Blight" sync /:Lahabrea:2B73:/
 840 "Dark IV"
 848 "Ultima" sync /:The Ultima Weapon:2B8B:/ window 50,10 jump 961
 
 
 ### Phase 5: Ultima ###
-1000 "--targetable--" sync /:2B87:The Ultima Weapon/ window 1000,100
+1000 "--targetable--" sync /:The Ultima Weapon:2B87:/ window 1000,100
```

Comment:
This isn't a typo, for what it's worth.

It's not obvious from timelines, but `:2B87:The Ultima Weapon` matches https://github.com/quisquous/cactbot/blob/master/docs/LogGuide.md#14-networkstartscasting, and the following `:The Ultima Weapon:2B87:` line on 1004 matches https://github.com/quisquous/cactbot/blob/master/docs/LogGuide.md#15-networkability